cmd/kyma/provision/gke/cmd.go

```diff
@@ -147,6 +150,18 @@ func (c *command) validateFlags() error {
 		errMessage.WriteString("\nRequired flag `credentials` has not been set.")
 	}
 
+	if len(strings.Split(c.opts.Location, "-")) <= 2 {
+		var answer bool
+		if !(c.opts.NonInteractive || c.opts.CI) {
+			answer = c.CurrentStep.PromptYesNo(fmt.Sprintf("Since you chose a region (%s) instead of a zone, %d number of nodes will be created on each zone in this region.\n"+
+				"You can also provide a different number of nodes or specify a zone instead.\n"+
+				"Are you sure you want to continue? ", c.opts.Location, c.opts.NodeCount))
+		}
+		if !(c.opts.NonInteractive || c.opts.CI) && !answer {
+			return fmt.Errorf("Aborting provisioning")
+		}
```

Comment:
```suggestion
		if !(c.opts.NonInteractive || c.opts.CI) {
			answer := c.CurrentStep.PromptYesNo(fmt.Sprintf("Since you chose a region (%s) instead of a zone, %d number of nodes will be created on each zone in this region.\n"+
				"You can also provide a different number of nodes or specify a zone instead.\n"+
				"Are you sure you want to continue? ", c.opts.Location, c.opts.NodeCount))
				
			if !answer {
				return fmt.Errorf("Aborting provisioning")
			}
		}
```

Reply:
🤦